Bedford-PD/bedford_complaints.yaml

```diff
@@ -0,0 +1,543 @@
+
+entityDefinitions:
+
+  person:
+    fqn: "general.person"
+    entitySetName: "MiddlesexBedfordPeople"
+    propertyDefinitions:
+      nc.SubjectIdentification:
+        type: "nc.SubjectIdentification"
+        column: "DEFENDANT_MSTR_NO"
+    conditions:
```

Comment:
You don't need to specify this condition.  It won't write anything if this is empty.

Reply:
I usually add conditions for all the columns no matter what, in case there's a ValueTransform now or in the future. Come to think of it, this needs a datasource!